src/components/Personality/PersonalityCard.jsx

```diff
@@ -99,6 +126,27 @@ class PersonalityCard extends Component {
                                         }
                                         href={`${this.props.hrefBase ||
                                             "personality/"}${personality._id}`}
+
+                                        style={{
+                                            background: "#2D77A3",
+                                            borderRadius: "30px",
+
+                                            position: "absolute",
+
+                                            height: "3em",
+                                            width: "8em",
+                                            paddingTop: "1em",
+
+                                            fontFamily: "Open Sans",
+                                            fontStyle: "normal",
+                                            fontWeight: "bold",
+                                            fontSize: "14px",
+                                            lineHeight: "107%",
```

Comment:
This is not needed. If you remove that you don't need to change width, height, and paddingTop.